delfin/db/api.py

```diff
@@ -841,3 +841,494 @@ def failed_task_delete_by_storage(context, storage_id):
+    """Update a masking view with the values dictionary."""
+    return IMPL.masking_views_update(context, values)
+
+
+def masking_views_delete(context, values):
+    """Delete multiple masking views."""
+    return IMPL.masking_views_delete(context, values)
+
+
+def masking_views_get(context, masking_view_id):
+    """Get a masking view or raise an exception if it does not exist."""
+    return IMPL.masking_views_get(context, masking_view_id)
+
+
+def masking_views_get_all(context, marker=None, limit=None,
+                          sort_keys=None,
+                          sort_dirs=None, filters=None, offset=None):
+    """Retrieves all masking views.
+
+    If no sort parameters are specified then the returned masking views are
+    sorted first by the 'created_at' key and then by the 'id' key in
+    descending order.
+
+    :param context: context of this request, it's helpful to trace the request
+    :param marker: the last item of the previous page, used to determine the
+                   next page of results to return
+    :param limit: maximum number of items to return
+    :param sort_keys: list of attributes by which results should be sorted,
+                      paired with corresponding item in sort_dirs
+    :param sort_dirs: list of directions in which results should be sorted,
+                      paired with corresponding item in sort_keys, for example
+                      'desc' for descending order
+    :param filters: dictionary of filters
+    :param offset: number of items to skip
+    :returns: list of masking views
+    """
+    return IMPL.masking_views_get_all(context, marker, limit, sort_keys,
+                                      sort_dirs, filters, offset)
+
+
+def masking_views_delete_by_storage(context, storage_id):
+    """Delete all the masking views of a device."""
+    return IMPL.masking_views_delete_by_storage(context, storage_id)
+
+
+def storage_host_grp_host_relations_create(context, values):
+    """Create a storage host grp host relation entry from the values
+    dictionary.
+    """
+    return IMPL.storage_host_grp_host_relations_create(context, values)
+
+
+def storage_host_grp_host_relations_update(context, values):
+    """Update a storage host grp host relation with the values dictionary."""
+    return IMPL.storage_host_grp_host_relations_update(context, values)
+
+
+def storage_host_grp_host_relations_delete(context, values):
+    """Delete multiple storage host grp host relations."""
+    return IMPL.storage_host_grp_host_relations_delete(context, values)
+
+
+def storage_host_grp_host_relations_get(context, host_grp_host_relation_id):
+    """Get a storage host grp host relation or raise an exception if it does
+    not exist.
+    """
+    return IMPL.storage_host_grp_host_relations_get(context,
+                                                    host_grp_host_relation_id)
+
+
+def storage_host_grp_host_relations_get_all(context, marker=None, limit=None,
```

Comment:
lets use table name as suffix for all operations storage_host_grp_host_rels_get_all ? applicable for all group relation related APS

Reply:
Updated